src/main/java/co/cask/hydrator/plugin/batch/HTTPToHDFSAction.java

```diff
@@ -59,17 +59,18 @@
 import javax.ws.rs.HttpMethod;
 
 /**
- * Action to fetch data from an external http endpoint and create a file in HDFS.
+ * Action to fetch data from an external http endpoint and create a file.
  */
 
 @Plugin(type = Action.PLUGIN_TYPE)
 @Name("HTTPToHDFS")
-@Description("Action to fetch data from an external http endpoint and create a file in HDFS.")
```

Comment:
Why change the description?
If its because the plugin is more generic than just HDFS, then also change the plugin name?